scripts/gradle/gradle-check.sh

```diff
@@ -53,19 +53,22 @@ if [ -z "$QUEUE_URL" ] || [ "$QUEUE_URL" != "null" ]; then
             RUNNING=$(curl -s -XGET ${WORKFLOW_URL}api/json | jq --raw-output .building)
         done
 
-        echo "Complete the run, checking results now......"
-        RESULT=$(curl -s -XGET ${WORKFLOW_URL}api/json | jq --raw-output .result)
+        if [ "$RUNNING" = "true" ]; then
+            echo "Timed out"
+            RESULT="TIMEOUT"
+        else
+            echo "Complete the run, checking results now......"
+            RESULT=$(curl -s -XGET ${WORKFLOW_URL}api/json | jq --raw-output .result)
+        fi
 
     fi
 fi
 
 echo "Please check jenkins url for logs: $WORKFLOW_URL"
-
+echo "Result: $RESULT"
 if [ "$RESULT" == "SUCCESS" ] || [ "$RESULT" == "UNSTABLE" ]; then
-    echo "Result: $RESULT"
     echo "Get codeCoverage.xml" && curl -SLO ${WORKFLOW_URL}artifact/codeCoverage.xml
-    echo 0
+    exit 0
```

Comment:
I'm assuming `echo 0` was a typo here, though it made no functional difference because the script would exit with a code of 0 anyway.

Reply:
It is not a typo it is just to print 0 to tell user it is downloading correctly.
Tho that is really a debug msg that kinda leftover.
We can delete this if needed.

Reply:
We can remove this line no need to reform the exit status to 0.